tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -836,17 +843,17 @@ async def get_last_block_id(self, client):
         msg = kvbc.SimpleKVBCProtocol.get_last_block_req()
         return kvbc.SimpleKVBCProtocol.parse_reply(await client.read(msg))
 
-    async def send_tracked_write(self, client, max_set_size):
+    async def send_tracked_write(self, client, max_set_size, long_exec=False):
         readset = self.readset(0, max_set_size)
         writeset = self.writeset(max_set_size)
         read_version = self.read_block_id()
-        msg = self.skvbc.write_req(readset, writeset, read_version)
+        msg = self.skvbc.write_req(readset, writeset, read_version, long_exec)
```

Comment:
How is this related to pre-execution?

Reply:
We have one test in the pre execution tests that build a message with long_exec